clipping_manager/views.py

```diff
@@ -50,20 +49,42 @@ def get_context_data(self, **kwargs):
         return ctx
 
     def get_queryset(self):
-        qs = Clipping.objects.select_related('book').for_user(user=self.request.user)
+        qs = Clipping.objects.select_related('book').for_user(user=self.request.user) \
+                                                    .exclude(deleted = True)
         self.filter = ClippingFilter(self.request.GET, request=self.request, queryset=qs)
         return self.filter.qs.distinct()
 
-
+class DeleteClipping(View):
+    http_method_names = ['post']
+
+    def post(self, request):
+        clipping_id = int(request.POST['clipping-id'])
+        clipping_to_delete = Clipping.objects.get(id=clipping_id)
+        clipping_book = clipping_to_delete.book
+        
+        # Clear eveything besides content_hash
+        # and update deleted status
```

Comment:
Maybe it makes sense to move this logic to a `delete` method on the model? Since we don't have any layer between models and views, fat models sound like an appropriate pattern for now.

Reply:
It's a great idea, but I don't want to completely override the model's delete method (it might come handy), so I'll move it to a completely new one - "clear"